network/policy/policy_windows.go

```diff
@@ -108,3 +126,235 @@ func SerializeOutBoundNATPolicy(policies []Policy, epInfoData map[string]interfa
 
 	return nil, fmt.Errorf("OutBoundNAT policy not set")
 }
+
+// GetPolicyType parses the policy and returns the policy type
+func GetPolicyType(policy Policy) CNIPolicyType {
```

Comment:
func SerializeHcnSubnetVlanPolicy(vlanID uint32) (vlanPolicySettingBytes []byte, err error) {


will declare the 2 return vars